java/code/src/com/redhat/rhn/frontend/xmlrpc/serializer/ErrataOverviewSerializer.java

```diff
@@ -60,9 +60,9 @@ protected void doSerialize(Object value, Writer output, XmlRpcSerializer seriali
         SerializerHelper helper = new SerializerHelper(serializer);
 
         helper.add("id", errata.getId());
-        helper.add("issue_date", errata.getIssueDate());
-        helper.add("date", errata.getUpdateDate());
-        helper.add("update_date", errata.getUpdateDate());
+        helper.add("issue_date", errata.getIssueDateIsoFormat());
+        helper.add("date", errata.getUpdateDateIsoFormat());
+        helper.add("update_date", errata.getUpdateDateIsoFormat());
```

Comment:
We better would return real date objects which are known by XMLRPC. But that would break all existing code which use this API.
So I think this is correct for now.